src/sentry/mail/adapter.py

```diff
@@ -133,6 +133,15 @@ def get_sendable_users(self, project):
         """
         return project.get_notification_recipients(self.alert_option_key)
 
+    def should_notify(self, group):
+        send_to = self.get_sendable_users(group.project)
+        if not send_to:
+            return False
+
+        if not group.is_unresolved():
```

Comment:
Could this be `return group.is_unresolved()` or will that complicate future changes?

Reply:
Oh yeah, will do that. There were just other things after it in the method I copied it from.